content/docs/introduction/roadmap.md

```diff
@@ -19,10 +19,11 @@ utilized in the Prometheus server. We plan on making use of this
 metadata in the future. The first step is to aggregate this data in-memory
 in Prometheus and provide it via an experimental API endpoint.
 
-### Prometheus metrics format as a standard
+### Adopt OpenMetrics
 
-We intend to submit a cleaned up version of our format for standardization
-to a group such as the IETF.
+The OpenMetrics working group is developing a new standard for metric exposition.
+We plan to adopt this format in our client libraries and Prometheus itself, so it
+can consume it from clients that do not support the current Prometheus format.
```

Comment:
This last clause doesn't seem quite right. Any client that exports it already likely supports the Prometheus format, and is going to pick it up because it's using our client libraries.

Reply:
Isn't the whole point that OM is an open format not tied to Prometheus. Going off of the assumption that the other end definitely uses our client libraries and supports our format makes the whole idea obsolete, no?

Reply:
It just reads oddly. I'd suggest adopt -> support and removing everything after the comma.